### ISSUE TITLE
[BUG] Unclosed bracket in SfTable with SFUI + vsf-default theme

### ISSUE THREAD

**Issue by slri** [vuestorefront/storefront-ui#1243]
**Describe the bug**
We're working on creating a new theme for Vue Storefront using Storefront UI and vsf-default theme as the base. Including storefront-ui and using it leads to a CssSyntaxError (Unclosed Bracket) on rebuild.

**How to reproduce**
1. Make a dummy theme out of vsf-default theme
2. Add SFUI as a dependency to default theme copy
3. Import any SFUI component into Home.vue (eg. `import { SfTabs } from '@storefront-ui/vue'`) and save
4. Console should log an error message in relation to SfTable.vue having an unclosed bracket

**Expected behavior**
SfTabs is available as a component and can be used.

**Actual behavior**
Following error pops up in the console: `CssSyntaxError: /var/www/node_modules/@storefront-ui/vue/src/components/organisms/SfTable/SfTable.vue:49:34: Unclosed bracket`

**Additional context**
Relevant tech: VSF 1.11.2, storefront-ui/shared 0.7.16, storefront-ui/vue 0.7.17

Solutions I've tried:
1. Copying vsf-capybara's package.json and webpack.config.js into the affected theme
2. Including `postcss-css-variables` as a plugin for postcss (suggested on a different project)

**Comment by AdamPawlinski** [comment]
The problem is in the postcss-flexbug-fixes plugin and was described in here https://github.com/luisrudge/postcss-flexbugs-fixes/issues/68. It should be resolved in the newest version of plugin 4.2.1, but upgrading doesn't work for me. I had to disable plugin to run these theme properly. You can do it inside vue-storefront/core/build/webpack.base.config.ts: 
`const postcssConfig = {
  loader: 'postcss-loader',
  options: {
    ident: 'postcss',
    plugins: (loader) => [
      // require('postcss-flexbugs-fixes'),
      require('autoprefixer')({
        flexbox: 'no-2009'
      })
    ]
  }
};` . 
@slri please let me know if upgrading works fine for you.

**Comment by slri** [comment]
Thanks for your response @AdamPawlinski.
The issue has changed for me so it no longer references that specific line. The error itself (Unclosed bracket), however, is the same.
However, commenting it out seems to work? Seems like the container needs a restart after the changes since it otherwise throws the same error regardless. After restarting the container, sfui is no longer seen by the app and therefore the app can't be built. `yarn list` acknowledges that sfui is installed, but it still can't find it.
For me the issue was fixed by running `yarn install --force --frozen-lockfile` after doing the things you mentioned in your response and restarting the containers.
The error honestly seems pretty erratic since I don't know how am I triggering it or how am I fixing it.

So to recap:
`docker-compose stop && docker-compose start` will keep the unclosed bracket persisting.
`docker-compose down && docker-compose start` will totally forget that sfui has even been installed, claiming it can't find it.
With your fixes, a restart of the containers, and `yarn install --force --frozen-lockfile` (for the app not being able to find sfui) the issue seems to be solved.

I can't give you more useful information at this time. It's working currently and I'm hoping it's because I commented out the line you mentioned. In the event that I get the time to troubleshoot it further, I'll give you all the info I gather.

The current stack:
@storefront-ui/shared@0.7.16
@storefront-ui/vue@0.7.17
on vsf 1.12.0

**Comment by andres-linares2211** [comment]
Any updates on this?

**Comment by Baroshem** [comment]
I created a project with current versions of SFUI and it isn't there. Closing the issue. If it appears again, the issue can be reopened.
